### DISCUSSION TITLE
How were the included models trained? (canny, depth, etc).

### DISCUSSION THREAD

**Discussion by whydna** [lllyasviel/ControlNet#226] [Q&A]
Anyone know if there is documentation/paper/info around the approach  used to train the default models? 

Looking to train my own and would be very useful. Things like data set size, diversity of data, what prompts, and other general details around general approach.

Thanks!

**Comment by geroldmeisinger** [comment]
I wrote an long article about control net training here https://civitai.com/articles/2078

**Comment by geroldmeisinger** [comment]
You can find more information about the original models here
[Adding Conditional Control to Text-to-Image Diffusion Models v2](https://arxiv.org/abs/2302.05543) - ControlNet paper v2
[Adding Conditional Control to Text-to-Image Diffusion Models v1](https://arxiv.org/abs/2302.05543v1) - original ControlNet paper
[ControlNet 1.0 repo](https://github.com/lllyasviel/ControlNet)
[ControlNet 1.1 repo](https://github.com/lllyasviel/ControlNet-v1-1-nightly)
